### PR TITLE
Update README.md to include region

### PR DESCRIPTION
**Pull request by art-tts**
Including region in example, if you're in Europe the example will not work, and will instead spit out errors.

### CODE REVIEW

**Comment by art-tts** [comment]
I signed it!

File: iot/mqtt_example/README.md [hunk 1]
```diff
@@ -46,6 +46,7 @@ run the sample as:
         --registry_id=my-registry \
         --device_id=my-node-device \
         --private_key_file=../rsa_private.pem \
+        --cloud_region=us-west1 \
```
Comment (gguuss) [inline iot/mqtt_example/README.md]: Nit. I'm not 100% sure we're in the us-west1 region yet, can you change this to `us-central1`, `europe-west1`, or `asia-east1`?

**Comment by gguuss** [comment]
@googlebot I signed it!

File: iot/mqtt_example/README.md (hunk 1 above)
Comment (art-tts) [reply iot/mqtt_example/README.md]: Did you verify?  I can modify.

Comment (gguuss) [reply iot/mqtt_example/README.md]: I did, `us-central1` is definitely supported, others I didn't list are not showing up for me.

**Comment by fhinkel** [comment]
@art-tts Thanks for the PR! Do you want to make the requested change?

**Comment by gguuss** [comment]
Closing, instructions must have been updated in a separate PR.

**Pull request closed without merging**